util/test/activate_chpl_test_venv.py

```diff
@@ -16,46 +16,56 @@ def error(message):
 def log(message):
     sys.stdout.write('[{0}]\n'.format(message))
 
-# Activate a virtualenv that has testing infrastructure requirements installed
+# Activate a venv that has testing infrastructure requirements installed
 def activate_venv():
     venv_dir = chpl_home_utils.get_chpl_test_venv()
     default_venv_dir = chpl_home_utils.get_chpl_venv()
 
     # user asserts that system already has the required dependencies installed:
     if venv_dir == 'none':
-        log('Skipping virtualenv activation because CHPL_TEST_VENV_DIR={0}. '
+        log('Skipping venv activation because CHPL_TEST_VENV_DIR={0}. '
             'test-venv requirements must be available.'.format(venv_dir))
 
     else:
         # using custom venv, does not check that our test requirements are met
         if venv_dir != default_venv_dir:
-            log('Using custom virtualenv because CHPL_TEST_VENV_DIR={0}. '
+            log('Using custom venv because CHPL_TEST_VENV_DIR={0}. '
                 'test-venv requirements must be available'.format(venv_dir))
 
         # check Chapel test-venv for successful installation sentinel
         else:
             sentinel_file = os.path.join(venv_dir, 'chpl-test-reqs')
             if not os.path.isfile(sentinel_file):
-                error('Chapel test virtualenv not available, run a top-level '
+                error('Chapel test venv not available, run a top-level '
                       '`{0} test-venv`'.format(chpl_make.get()))
 
-        activation_file = os.path.join(venv_dir, 'bin', 'activate_this.py')
-        if not os.path.isfile(activation_file):
-            error('Activation file {0} is missing'.format(activation_file))
+        # "activate" the venv
+        # this is based upon activate_this.py from virtualenv.
+        old_os_path = os.environ.get('PATH', '')
 
-        old_path = os.environ.get('PATH', '')
-
-        # actually activate
-        with open(activation_file) as f:
-            code = compile(f.read(), activation_file, 'exec')
-            exec(code, dict(__file__=activation_file))
-
-        # put PATH back so that the system python may be used.
+        # do not change PATH so that the system python may be used.
         # python scripts using the virtualenv dependencies should use
         #
         #   from test import activate_chpl_test_venv
         #
         # near to the 'import' of a virtual dependency.
-        os.environ['PATH'] = old_path
+
+        #os.environ['PATH'] = os.path.dirname(os.path.abspath(__file__)) + os.pathsep + old_os_path
```

Comment:
leftover debug?

Reply:
This is something that the `activate_this.py` did but then we would un-do. I am open to suggestions for whether or not it is worth leaving in as a comment.

Reply:
Ah, reading the whole file with the comment in context that makes sense (I was only looking at the diff and it seemed like an unintentional comment)